### DISCUSSION TITLE
Beginner trader

### DISCUSSION THREAD

**Discussion by stefanolucia2019** [community/community#50218] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Good day 
I don't see any longer my assets (CRO) in the crypto.com defi **desktop** wallet. I can see them only in Crypto.org Explorer but I am no longer able to manage them. I don't know if this is the right place to ask because I am not expert. Can anybody address me to the right place to have any suggestion ? The customer care support of crypto.com doesn't reply or is not useful. Thank you. 

Stefano.

**Comment by Anmol-Baranwal** [comment, marked as answer]
There may be a few possible reasons for this:

- It's possible that your wallet is not properly synchronized with the blockchain, which may be preventing you from seeing your assets. Try checking if your wallet is fully synced and try again.

- Make sure that you have selected the correct network (mainnet or testnet) in the wallet. If you have selected the wrong network, you may not be able to see or manage your assets.

- If you recently made a transaction with your assets, it's possible that the transaction has not yet been confirmed by the blockchain. In this case, you may need to wait for the transaction to be confirmed before you can see or manage your assets.

**Reply by stefanolucia2019** [reply]
Hello Anmol-Baranwal

Thank you for your kind reply. I think none of the cases above apply to my issue. I just checked once again and the wallet is on mainnet and it's been a long time I don't make transactions. I also think that my wallet is (or was) syncronized because until last October everything was working properly. 

Suddenly I couldn't see anymore the assets in the wallet but only in Crypto.org. Explorer. I don't know if I unvolountary done something what I can add is that the wallet is protected under a Ledger device. Before I could click on its address and see the assets but this is no longer possible, Could any firewall be the reason for things not working ?  

How should I check if my wallet is still syncronized with the block chain ? I am reading articles on Crypto.com support but I am not even sure of how to proceed. I am quite sure that I am using the latest version of the wallet (v1.4.1). Thank you again for the moment.

Regards
Stefano.

**Reply by Anmol-Baranwal** [reply]
Based on your description, it is possible that your wallet is not synchronized with the blockchain, which is why you are unable to see the assets in the wallet. However, it is also possible that there could be other reasons for this issue.

To check if your wallet is still synchronized with the blockchain, you can try these:

1. Look for any status messages or alerts in the wallet interface that could indicate synchronization issues.

2. You can use a blockchain explorer to check the status of your wallet address and the transactions associated with it. If you see your recent transactions on the explorer, it means that your wallet is synchronized with the blockchain.

3. Sometimes refreshing the wallet or resetting it can help to resolve synchronization issues.

Regarding the firewall question, it is unlikely that a firewall would prevent your wallet from syncing with the blockchain. However, it is always a good idea to ensure that any firewalls or security software are not blocking the wallet application.

> If none of these help, you may want to consider waiting for others to respond, as they could potentially provide a better solution.

**Reply by Anmol-Baranwal** [reply]
@stefanolucia2019 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by stefanolucia2019** [reply]
Good day Anmol-Baranwal

Thanks again for your message; I think I marked your answer correctly. 

I can confirm that I see my desktop wallet in Crypto.org Explorer and also the last operation done has a confirmed status. Perhaps the problem is with the Ledger wallet that I had associated with the desktop wallet address... The only thing I know is that up to a certain point I had access to my assets by simply clicking on the Ledger wallet address (in the desktop wallet) but now nothing happens. 

I also tried to unstake other CROs but the desktop wallet tells me that the address does not match the Ledger wallet address. In fact, the application I have on Ledger Live has a different address... I wrote to Ledger support but after the usual automatic reply etc. etc. they disappeared and never answered me again. 

I think that the solution to the problem is not difficult and far away but I can't find it by myself. I hope that others can write something useful because I have no idea how to proceed. Thank you for the moment. 

Kind regards
Stefano.

**Reply by stefanolucia2019** [reply]
p.s. I made the staking directly in the Crypto.com desktop wallet and not with Ledger Live as it is also possible.

**Reply by Anmol-Baranwal** [reply]
I would strongly suggest you to unmark my answer if you want others to answer this.

Since no one would answer once this is marked. Else, it is your choice.
Thanks

**Reply by stefanolucia2019** [reply]
Thank you I just unmarked your answer. The issue is still going on....
Thanks.

**Reply by stefanolucia2019** [reply]
Hello Anmol-Baranwal
Issue is finally sorted :-) Thank you again. 
Regards
Stefano.

**Reply by Anmol-Baranwal** [reply]
Okay, thanks for letting me know.
